### DISCUSSION TITLE
Transparent Filter Popup

### DISCUSSION THREAD

**Discussion by codygabhart** [macgile/DataGridFilter#177] [Q&A]
Hello! 

The filter menu/popup is transparent and I cannot find a way to change it. Has anyone else come across this and discovered a solution?

![image](https://github.com/user-attachments/assets/675f8481-1aaa-4db4-8a7b-1d01fedce812)

**Comment by macgile** [comment]
Hi, 
Read the documentation
Best regard

**Reply by codygabhart** [reply]
Sorry! I finally found the correct property. Thank you!
